run-as/run-as.c

```diff
@@ -122,17 +122,19 @@ int main(int argc, char **argv)
     }
 
     /* retrieve package information from system */
+    errno = 0;
     pkgname = argv[1];
     if (get_package_info(pkgname, &info) < 0) {
-        panic("Package '%s' is unknown\n", pkgname);
+        panic("Package '%s' is unknown, %s\n", pkgname, strerror(errno));
```

Comment:
It will print here string representation of `errno`, right? but it's 0.

Reply:
if `get_package_info()` fails `errno` will be non-zero